addons/web/static/src/model/relational_model/relational_model.js

```diff
@@ -392,46 +392,58 @@ export class RelationalModel extends Model {
         }
         const proms = [];
         let nbOpenGroups = 0;
+
+        const groupsInfo = [];
         for (const group of groups) {
+            const groupInfo = {};
```

Comment:
do this the other way around: keep group for the group info, and rename group/groups here to groupData or something